### PR TITLE
Added more ebusd sensors.

### PR DESCRIPTION
**Pull request by sashao**
**Description:**

Pull request in home-assistant https://github.com/home-assistant/home-assistant/pull/26750

## Checklist:

- [+] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [+] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/ebusd.markdown [hunk 1]
```diff
@@ -42,7 +45,7 @@ name:
   required: false
   default: ebusd
 monitored_conditions:
-  description: List of condition to monitor.
+  description: List of condition to monitor. Refer to https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/ebusd/const.py for device support table.
```
Comment (klaasnicolaas) [inline source/_components/ebusd.markdown]: Make it possible to click on the url

File: source/_components/ebusd.markdown [hunk 2]
```diff
@@ -20,6 +20,9 @@ Enable the sensor by adding the following to your `configuration.yaml` file:
 ebusd:
   host: 127.0.0.1
   circuit: '700'
+  monitored_conditions:
```
Comment (klaasnicolaas) [inline source/_components/ebusd.markdown]: Only required variables in the first example

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/ebusd.markdown (hunk 2 above)
Comment (sashao) [reply source/_components/ebusd.markdown]: Thanks. Removed.

File: source/_components/ebusd.markdown (hunk 1 above)
Comment (sashao) [reply source/_components/ebusd.markdown]: Fixed. Thanks.

Comment (klaasnicolaas) [reply source/_components/ebusd.markdown]: I've just talked to @frenck, and we think that it's better to omit the link 😉

Comment (sashao) [reply source/_components/ebusd.markdown]: Yes, that was too technical. Fixed by adding explanation to monitored_conditions.

File: source/_components/ebusd.markdown [hunk 3]
```diff
@@ -42,7 +42,7 @@ name:
   required: false
   default: ebusd
 monitored_conditions:
-  description: List of condition to monitor.
+  description: List of condition to monitor. Note that not all monitored_conditions listed here can be supported by your circuit. This integration maps limited set of keys to circuit specific ebusd values.
```
Comment (MartinHjelmare) [inline source/_components/ebusd.markdown]: List of conditions to monitor.

**Review by MartinHjelmare — COMMENTED** [review]
Just a grammar change suggested.

File: source/_components/ebusd.markdown (hunk 3 above)
Comment (sashao) [reply source/_components/ebusd.markdown]: Here I have tried to explain complications which I met during first attempt to setup this integration.
There is changing map of supported sensors per circuit:
https://github.com/sashao/home-assistant/blob/dev/homeassistant/components/ebusd/const.py

**Comment by sashao** [comment]
I am happy to admit that code is merged to dev branch! And I am ready to fix documentation issues as quickly as possible.
Please help me as I am not a native speaker.

File: source/_components/ebusd.markdown (hunk 3 above)
Comment (MartinHjelmare) [reply source/_components/ebusd.markdown]: Yes, that's good. Just change the first sentence as my suggestion.

**Review by MartinHjelmare — APPROVED** [review]

File: source/_components/ebusd.markdown (hunk 3 above)
Comment (sashao) [reply source/_components/ebusd.markdown]: Fixed.

**Review by frenck — APPROVED** [review]
Looks good @sashao! 🎉

**Pull request merged**